src/ScClient.ts

```diff
@@ -15,6 +15,12 @@ export interface Response<T = any> {
   status: boolean;
   event: boolean;
   payload: T;
+  errors: string | Error[];
```

Comment:
May it be a string? If so our error handling with pushing them to array is wrong.

Reply:
Oh, yes. You are rights. I'll fix it